README.md

```diff
@@ -176,4 +176,6 @@ browserWindow.LastName.Text ="Doe";
 
 // Click on the Save button
 browserWindow.Save.Click();
-```
\ No newline at end of file
+```
+
+Follow the steps from [this guide](http://www.codewrecks.com/blog/index.php/2011/06/06/witch-version-of-browser-is-used-by-the-webbrowser-control/) to use a more recent version of IE in the Object Recorder.
```

Comment:
My point is that the WebBrowser control from the CUITe Object Recorder was using an older version of IE. The page for which I was 'recording' elements was not compatible with that version so I followed the steps from that article to make the Object Recorder use the latest version of IE that is available on my system.

Probably we should add a .reg file with the key and value that is placed inside the packages folder where the CUITe Object Recorder resides.